editoast/src/views/test_app.rs

```
@@ -106,16 +108,12 @@ impl TestAppBuilder {
         };
 
         // Setup tracing
-        let sub = tracing_subscriber::fmt()
-            .pretty()
-            .with_env_filter(
-                tracing_subscriber::EnvFilter::builder()
-                    .with_default_directive(tracing_subscriber::filter::LevelFilter::DEBUG.into())
-                    .from_env_lossy(),
-            )
-            .with_span_events(tracing_subscriber::fmt::format::FmtSpan::CLOSE)
-            .finish();
-        let tracing_guard = tracing::subscriber::set_default(sub);
+        let exporter = opentelemetry_stdout::SpanExporter::default();
+        let telemety_config = TelemetryConfig {
+            telemetry_kind: TelemetryKind::Opentelemetry,
+            ..Default::default()
+        };
+        let _ = try_init_tracing(EditoastMode::Webservice, &telemety_config, exporter);
```

Errors shouldn't be silently ignored even in tests. Maybe `expect` or we should at least log something.

---

Done.